ui/src/components/footer/FooterMobile.js

```diff
@@ -0,0 +1,304 @@
+          <VStack w={'80%'}>
+            <Link href="/" pb="7">
+              <Image
+                h="6.5vh"
+                src={process.env.PUBLIC_URL + '/images/ripple.svg'}
+                alt="Logo of HMCC"
+              />
+            </Link>
+            <Box>
+              <Text fontWeight="700" align="center">
+                Harvest Mission Community Church
+              </Text>
+              <Text fontWeight="700" align="center">
+                (Hong Kong)
+              </Text>
+            </Box>
+            <Text color="white" fontWeight="600">
+              Today's BRP : <DailyBrp />
+            </Text>
+            <Button
+              size="sm"
+              variant="outline"
+              colorScheme="white"
+              as={Link}
+              href="https://hmcc.tv"
+            >
+              <Text px="7" fontWeight="700">
+                HMCC.TV
+              </Text>
+            </Button>
+          </VStack>
+          <Accordion w="100%" allowToggle>
+            <AccordionItem borderTop="1px">
+              {({ isExpanded }) => (
+                <>
+                  <h2>
+                    <AccordionButton>
+                      <Box as="span" flex="1" textAlign="left" fontWeight="700">
+                        About
+                      </Box>
+                      {isExpanded ? (
+                        <MinusIcon fontSize="12px" />
+                      ) : (
+                        <AddIcon fontSize="12px" />
+                      )}
+                    </AccordionButton>
+                  </h2>
+                  <AccordionPanel pb={4}>
+                    <Accordion>
+                      <AccordionItem border="none">
+                        <h2>
+                          <AccordionButton
+                            as={Link}
+                            href="/about-us#our-story"
+                            border="none"
+                          >
+                            <Box as="span" flex="1" textAlign="left">
+                              Our Story
+                            </Box>
+                          </AccordionButton>
+                        </h2>
+                      </AccordionItem>
+                      <AccordionItem border="none">
+                        <h2>
+                          <AccordionButton
+                            as={Link}
+                            href="/about-us#vision-mission"
+                          >
+                            <Box as="span" flex="1" textAlign="left">
+                              Vision & Mission
+                            </Box>
+                          </AccordionButton>
+                        </h2>
+                      </AccordionItem>
+                      <AccordionItem border="none">
+                        <h2>
+                          <AccordionButton as={Link} href="/about-us#values">
+                            <Box as="span" flex="1" textAlign="left">
```

Comment:
all these accordions is repetitive. can make it shorter and cleaner by mapping like an array
``` { array.map... <AccordionItem> ... </AccordionItem>)```